plugin/valgrind.vim

```diff
@@ -123,8 +123,8 @@ function! s:Valgrind( ... )
     else
         silent execute 'split '.l:tmpfile
     endif
-    silent execute 'g!/^==\d*==/d'
-    silent execute '%s/^==\d*== //e'
+    silent execute 'g!/^\(\d\+: \)\===\d*==/d'
```

Comment:
I agree the use case is valid, but I'm not sure hard-coding the optional prefix here is the right answer - would we be better off having a prefix variable which can be configured in a vimrc?